simplq/src/components/pages/Admin/AdminPage.jsx

```diff
@@ -16,8 +19,31 @@ import getToursteps from './getTourSteps';
 const TIMEOUT = 10000;
 let timeoutId;
 
-export default (props) => {
+const isQueueDeleted = (queueInfo) => queueInfo?.status === 'DELETED';
+
+const AdminPage = (props) => {
   const queueId = props.match.params.queueId;
+  const queueInfo = useSelector(selectQueueInfo);
+  const dispatch = useDispatch();
+  const history = useHistory();
+  const getQueueInfo = useCallback(useGetQueueInfo(), []);
+
+  useEffect(() => {
+    dispatch(getQueueInfo({ queueId }));
+  }, [dispatch, queueId, getQueueInfo]);
```

Comment:
We don't need to do this, I think getQueueInfo is already periodically called.

Reply:
I did it for a different reason. Removing it in #638. 
I thought not just back button, but if any user lands on that url ( queue/queueId ) and Redux store does not know about this queue, I'd make sure we fetch it from server and then check.